### ISSUE TITLE
(node:26972) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0'

### ISSUE THREAD

**Issue by tonissimo** [dsandor/dymojs#16]
I am using DymoJS to print labels on my local machine and I get this error:

(node:26972) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.

I tried installing ssl-root-cas (https://www.npmjs.com/package/ssl-root-cas) but that does not help. I also tried changing NODE_TLS_REJECT_UNAUTHORIZED to '1' in dymojs's index.js but it does not work either.

**Comment by cpgb85** [comment]
Same issue here. Ever figure this out?

**Comment by tonissimo** [comment]
Yes, for me what solved the issue was to put the first line of the XML label on the same line as the opening backtick. (I previously had the first XML line on the next line). I hope it helps!

<img width="306" alt="Screen Shot 2021-03-25 at 3 48 54 PM" src="https://user-images.githubusercontent.com/33302447/112534611-a4327580-8d81-11eb-9ae5-4e652280f26d.png">

**Comment by einaraglen** [comment]
I did what @tonissimo suggested, and it did not seem to change anything, I then moved the line :

`dymo.print("DYMO LabelWriter Wireless", labelXml);`

And placed it under the string XML, and that seem to work just fine! it is printing now!


EDIT: and I also changed name of Printer as I am using the Wireless verison

**Comment by kardall** [comment]
> Yes, for me what solved the issue was to put the first line of the XML label on the same line as the opening backtick. (I previously had the first XML line on the next line). I hope it helps!
> 
> <img alt="Screen Shot 2021-03-25 at 3 48 54 PM" width="306" src="https://user-images.githubusercontent.com/33302447/112534611-a4327580-8d81-11eb-9ae5-4e652280f26d.png">

This is the reason that I love github...

Totally a different error message, but I decided to look at what the newest error was since it really hasn't been maintained well. This XML string is what breaks the new 8.7.2+ Drivers...

Was having CORS problems and all that... as soon as I removed the linefeed after the ` the problem resolved itself... amazing...
